packages/cssnano/package.json

```diff
@@ -24,7 +24,8 @@
     ],
     "license": "MIT",
     "dependencies": {
-        "cosmiconfig": "^7.0.0",
+        "lilconfig": "^2.0.3",
+        "yaml": "^1.10.2",
```

Comment:
I was uncertain whether to keep YAML config parsing since it was never documented explicitly, but removing would be a breaking change in theory.

Reply:
Sounds like a good call to keep it for now, even if the chance of it being used is remote 👍 

Question:

Since the tests you added were paired with a `.cssnanorc.json` and you mentioned keeping YAML support, I decided to attempt running the test suite with a `.cssnanorc.yaml` and a few other variations (replacing `.cssnanorc.json` in the `config_loading` dir) for the sake of diligence. And it doesn't appear to be picked up.

It's entirely possible I'm having some form of "brainfart" and doing something wrong here - but shouldn't that work?

(Out of curiosity I had a look at the `searchItems` found by lilconfig in `lilconfigSync.search()` and it only appears to look for `package.json`, `cssnanorc.json`, `cssnano.config.js`, and `cssnanorc.js`, `.cssnanorc.cjs`, and `cssnano.config.cjs`)

Reply:
My inner conviction is nobody is really using the YAML format, so I did not bother checking whether it works, I have just copied the instructions on the lilconfig repo. Looking at the `lilconfig` code, it seems it does not even load `.rc` (with no extension) by default.